INSTALL.md

```diff
@@ -302,96 +302,23 @@ AWS_ACCESS_KEY_ID=<ACCESS_KEY> AWS_SECRET_ACCESS_KEY=<SECRET_KEY> python3 script
-
-```sh
-# Find ELASTIC_SEARCH_KIBANA_USER_POOL_APP_CLIENT_ID in the printout
-serverless info --verbose
-
-# Create new user
-aws cognito-idp sign-up \
-  --region <REGION> \
-  --client-id <ELASTIC_SEARCH_KIBANA_USER_POOL_APP_CLIENT_ID> \
-  --username <youremail@address.com> \
-  --password <TEMP_PASSWORD> \
-  --user-attributes Name="email",Value="<youremail@address.com>"
-
-# Find ELASTIC_SEARCH_KIBANA_USER_POOL_ID in the printout
-# Notice this is a different ID from the one used in the last step
-serverless info --verbose
-
-# Confirm new user
-aws cognito-idp admin-confirm-sign-up \
-  --user-pool-id <ELASTIC_SEARCH_KIBANA_USER_POOL_ID> \
-  --username <youremail@address.com> \
-  --region <REGION>
-
-# Example
-aws cognito-idp sign-up \
-  --region us-west-2 \
-  --client-id 4rcsm4o7lnmb3aoc2h64nv1324 \
-  --username jane@amazon.com \
-  --password Passw0rd! \
-  --user-attributes Name="email",Value="jane@amazon.com"
-
-aws cognito-idp admin-confirm-sign-up \
-  --user-pool-id us-west-2_sOmeStRing \
-  --username jane@amazon.com \
-  --region us-west-2
+#### Store Integration Transform Info in AWS System Manager Parameter Store 
+FHIR Works on AWS will need to know the URL and AWS region of the Integration Transform that you would like to send requests to. Please specify the URL and AWS region by running this command with the correct values filled in for the placeholders.
 ```
+aws ssm put-parameter --region $region --cli-input-json \
+'{"Type": "String", "Name": "/fhir-service/integration-transform/<STAGE>/url", "Value": "<IntTranUrl>"}'
 
-###### Get Kibana url
-
-After the Cognito user is created and confirmed you can now log in with the username and password, at the ELASTIC_SEARCH_DOMAIN_KIBANA_ENDPOINT (found with the `serverless info --verbose` command). **Note** Kibana will be empty at first and have no indices, they will be created once the FHIR server writes resources to the DynamoDB
-
-#### DynamoDB table backups
-
-Daily DynamoDB Table back-ups can be optionally deployed via an additional 'fhir-server-backups' stack. The installation script will deploy this stack automatically if indicated during installation.
+aws ssm put-parameter --region $region --cli-input-json \
+'{"Type": "String", "Name": "/fhir-service/integration-transform/<STAGE>/awsRegion", "Value": "<AWS-REGION>"}'
 
-The reason behind multiple stacks is that backup vaults can be deleted only if they are empty, and you can't delete a stack that includes backup vaults if they contain any recovery points. With separate stacks it is easier for you to operate.
 
-These back-ups work by using tags. In the [serverless.yaml](./serverless.yaml) you can see ResourceDynamoDBTableV2 has a `backup - daily` & `service - fhir` tag. Anything with these tags will be backed-up daily at 5:00 UTC.
+Exp

```

Comment:
Exp?

Reply:
It's supposed to be an abbreviation for example, I'll change it to be `E.g.` to be clearer what it means